react/components/ProductImages/components/Carousel/index.js

```diff
@@ -259,7 +259,7 @@ class Carousel extends Component {
             bulletSelector: `.swiper-pagination-bullet`,
             bulletClass: `swiper-pagination-bullet ${cssHandles.swiperBullet}`,
             bulletActiveClass:
-              'c-action-primary swiper-pagination-bullet-active',
+              `c-action-primary swiper-pagination-bullet-active ${cssHandles.swiperBullet}--isActive`,
```

Comment:
You could use [applyModifiers](https://github.com/vtex-apps/css-handles) here

Reply:
Sorry, I'm new to VTEX IO, but is resolved. Thanks.